.gitignore

```diff
@@ -11,7 +11,6 @@ frontend/reports
 frontend/strykerTmp
 /.stryker-tmp
 
-
```

Comment:
Even though this is just a new line, you dont want to be merging changes to any file that is not supposed to change.

Reply:
Solved!